### PR TITLE
Response to comments from CRAN `0.4.0-rc2`

### LINKED ISSUE #283: CRAN comments
Comments from CRAN reviewer.

> The Description field is intended to be a (one paragraph) description of
what the package does and why it may be useful. Please add more details
about the package functionality and implemented methods in your
Description text.

>If there are references describing the methods in your package, please
add these in the description field of your DESCRIPTION file in the form
authors (year) <doi:...>
authors (year) <arXiv:...>
authors (year, ISBN:...)
or if those are not available: <https:...>
with no space after 'doi:', 'arXiv:', 'https:' and angle brackets for
auto-linking.
(If you want to add a title as well please put it in quotes: "Title")

>You have examples for unexported functions. Please either omit these
examples or export these functions.
Examples for unexported function
   default_cdisc_join_keys() in:
      col_labels.Rd

>Please fix and resubmit.

### PR DESCRIPTION
**Pull request by averissimo**
Closes #283

#### Changes description

- [x] Proposes a new description
- [ ] Remove/adress `@examples` of non-exported functions

#### Here are a few alternatives

1. Utilities to provide a unified data model for `teal` applications focusing on reproducibility and relational data.
    
1. Utilities to provide a unified data model for `teal` applications focusing on reproducibility. The model and data transformation functions support a wide range of data structures, such as `data.frame`, `MultiAssayExperiment`, matrices and others. Relationships between datasets can be modelled using `join_keys` function.
1. Utilities to provide a unified data model for `teal` applications focusing on reproducibility that supports a wide range of data structures, such as `data.frame`, `MultiAssayExperiment`, matrices and others.
1. Utilities to provide a unified data model for `teal` applications focusing on reproducibility. `teal.data` supports a wide range of data structures, such as `data.frame`, `MultiAssayExperiment` and matrices.
1. With a core focus on reproducibility, these utilities provide a unified data model for teal applications, supporting a diverse range of data structures such as `data.frame`, `MultiAssayExperiment`, matrices, and others.
1. Utilities to provide a unified data model for `teal` applications focusing on reproducibility and relational data.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -19,8 +19,8 @@ Authors@R: c(
     person("Dony", "Unardi", , "unardid@gene.com", role = "rev"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
   )
-Description: Utilities for data transformations compatible with 'teal'
-    applications.
+Description: Utilities to provide a unified data model for `teal` applications
```
Comment (gogonzo) [inline DESCRIPTION:22]: 👍

Comment (averissimo) [reply DESCRIPTION:22]: This is not that different from what we had, I phrased similarly to `tidyverse` descriptions, but I'm thinking it doesn't address part of the comment

> Please add more details about the package functionality and \***implemented methods**\* in your
Description text. (emphasis mine)

Comment (averissimo) [reply DESCRIPTION]: Small tweak to mention the `teal_data` class in the spirit of `tibble`

> Description: Provides a 'tbl_df' class (the 'tibble') with stricter checking and better formatting than the traditional data frame.

```diff
-Utilities to provide a unified data model for teal applications focusing on reproducibility and relational data.
+Provides a `teal_data` class as a unified data model for `teal` applications focusing on reproducibility and relational data.
```

Comment (chlebowa) [reply DESCRIPTION]: unified data _container_?

Comment (averissimo) [reply DESCRIPTION]: Good & accurate suggestion :smile: 

I wrote _data model_ as it supports a relational model with `join_keys` (and it's a nice buzzword). I'm not opposed to change it to container though

**Review by gogonzo — APPROVED** [review]

File: DESCRIPTION [hunk 2]
```diff
@@ -19,8 +19,8 @@ Authors@R: c(
     person("Dony", "Unardi", , "unardid@gene.com", role = "rev"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
   )
-Description: Utilities for data transformations compatible with 'teal'
-    applications.
+Description: Provides a `teal_data` class as a unified data model for `teal` applications
```
Comment (pawelru) [inline DESCRIPTION:22]: I have seen somewhere that "`" is forbidden. Better to come back to the original quotes.

Comment (pawelru) [reply DESCRIPTION:22]: None of the available packages has "`" whereas there are a lot of with "'".

(I just did ctrl+f on https://cran.r-project.org/web/packages/available_packages_by_name.html)

Comment (averissimo) [reply DESCRIPTION:22]: Noted. Thanks for this. I am so used to backticks that I didn't even think twice about the DESCRIPTION file

This change was done in 75d07c203b2fc92b36e31ca8fe98d23b8e226714 and resubmitted 2 days ago

**Pull request merged**